src/tests/SortingProducts.spec.js

```diff
@@ -14,7 +14,7 @@ test.beforeEach(async ({ app }) => {
 test.describe('Saucedemo tests for sorting products on invertory page', () => {
     test('Sorting products by name A-Z', async ({ app }) => {
         const productNamesBeforeSort = await app.inventory.inventoryItems.allTextContents();
-        await app.inventory.sortProductNameAZ();
+        await app.inventory.performProductSort('az');
 
         const productNamesAfterSortAZ = await app.inventory.inventoryItems.allTextContents();
         const sortedNamesAZ = [...productNamesBeforeSort].sort();
```

Comment:
Please, move this to the helper. Along with lines 29, 38, and 47 to make your code cleaner